packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,47 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  &:not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: var(--gcds-link-underline-offset);
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-link-decoration-thickness);
+  }
```

Comment:
I just noticed that we still had the check for disabled in here as a leftover from the button css. Let's remove it since it's not needed. Suggesting to change line 2-10 to this:

```suggestion
 cursor: pointer;
 color: var(--gcds-link-default);
 text-decoration: underline;
 text-underline-offset: var(--gcds-link-underline-offset);
 text-decoration-color: currentColor;
 text-decoration-thickness: var(--gcds-link-decoration-thickness);
```

Reply:
Good catch!